.github/workflows/main.yml

```diff
@@ -64,23 +64,21 @@ jobs:
   cypress_tests:
     name: Cypress tests
     runs-on: ubuntu-latest
-    needs: install_dependencies
+    strategy:
+      fail-fast: false
+      matrix:
+        browser: [chrome, firefox]
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
-        with:
-          node-version: "14"
-      - name: Restore cache
-        uses: actions/cache@v2.1.4
-        with:
-          path: "**/node_modules"
-          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
       - name: Run Cypress tests
         uses: cypress-io/github-action@v2
         with:
-          start: npm run dev
+          start: npm run start
+          build: npm run build
```

Comment:
@Ninamma Really liked this as we can build the application before running the tests! See docs: https://github.com/cypress-io/github-action#build-app